### PR TITLE
loader prototype

### PR DESCRIPTION
**Pull request by ALI-KHALED-13**

### CODE REVIEW

File: client/src/pages/Login/index.tsx
Comment (Elena-Erganyan) [inline client/src/pages/Login/index.tsx:1]: We can delete `isLoading` (line 61) because don't use it anymore

File: client/src/pages/Register/index.tsx
Comment (Elena-Erganyan) [inline client/src/pages/Register/index.tsx:1]: Same, line 66, remove `isLoading`

**Review by Elena-Erganyan — CHANGES_REQUESTED** [review]
I like how it looks, beautiful 😃 
Couple of comments, very small ones

File: client/src/GlobalStyles.ts [hunk 1]
```diff
@@ -87,6 +87,7 @@ const GlobalStyles = createGlobalStyle`
     font-size: 62.5%;
     scroll-behavior: smooth;
     background-color: ${colors.lightGray};
+    color-scheme: light dark;
```
[the remark this answers is not in the document]
Comment (ALI-KHALED-13) [reply client/src/GlobalStyles.ts:90]: so this seems to help to display the website correctly if a user has a "dark reader" extension turned on for our website, I noticed it works with my trivia game so transferred it here

**Review by Elena-Erganyan — APPROVED** [review]

**Pull request merged**